kas/include/local.yml

```diff
@@ -0,0 +1,31 @@
+header:
+  version: 14
+
+local_conf_header:
+  default: |
+    # -- Default local.conf from meta-poky/conf/local.conf.sample -- #
+    EXTRA_IMAGE_FEATURES ?= "debug-tweaks"
+    USER_CLASSES ?= "buildstats"
+    PATCHRESOLVE = "noop"
+    BB_DISKMON_DIRS ??= "\
+        STOPTASKS,${TMPDIR},1G,100K \
+        STOPTASKS,${DL_DIR},1G,100K \
+        STOPTASKS,${SSTATE_DIR},1G,100K \
+        STOPTASKS,/tmp,100M,100K \
+        HALT,${TMPDIR},100M,1K \
+        HALT,${DL_DIR},100M,1K \
+        HALT,${SSTATE_DIR},100M,1K \
+        HALT,/tmp,10M,1K"
+    PACKAGECONFIG:append:pn-qemu-system-native = " sdl"
+    PACKAGE_CLASSES = "package_rpm"
+    SDKMACHINE = "x86_64"
+    CONF_VERSION = "2"
+
+    DL_DIR = "/home/talel/Desktop/YoctoShare/downloads"
+    SSTATE_DIR = "/home/talel/Desktop/YoctoShare/sstate-cache"
```

Comment:
these are hardcoded paths. Do we need them ?

Reply:
I remember removing them in this commit: https://github.com/riscv/meta-riscv/pull/433/commits/ddb39302875d7efdb7d8dec127e78a091440c6cc
I will check again.